File: app/src/main/java/com/thesunnahrevival/sunnahassistant/views/WelcomeActivity.java
package com.thesunnahrevival.sunnahassistant.views;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.thesunnahrevival.sunnahassistant.R;

public class WelcomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcome);
    }

    public void openSettings(View v) {
        QuickSetupFragment quickSetupFragment = new QuickSetupFragment();
        quickSetupFragment.show(getSupportFragmentManager(), "setup");
    }

}

File: app/src/main/java/com/thesunnahrevival/sunnahassistant/views/WelcomeActivity.kt
package com.thesunnahrevival.sunnahassistant.views

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.thesunnahrevival.sunnahassistant.R

class WelcomeActivity : AppCompatActivity(), View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcome)
    }

    override fun onClick(v: View) {
        startActivity(Intent(this,MainActivity::class.java))
    }
}